application/features/Audio.py

```
@@ -103,26 +110,31 @@
         exit_status, _, stdout, _ = self.audio.exec_command_blocking(load_module_command)
         if exit_status != 0:
-            print(f'AudioWebSocket: audio_id={audio_id}: unable to load pactl module-null-sink sink_name={sink_name}')
+            logger.warning(f'AudioWebSocket: audio_id={audio_id}: unable to load pactl module-null-sink sink_name={sink_name}')
             return
         load_module_stdout_lines = stdout.readlines()
+        logger.debug("AudioWebSocket: Load Module: {}".format(load_module_stdout_lines))
         self.module_id = int(load_module_stdout_lines[0])
 
         keep_launching_ffmpeg = True
 
         def ffmpeg_launcher():
```

Ruff

Missing return type annotation for private function `ffmpeg_launcher`

Add return type annotation: `None`

(ANN202)